app/gui2/src/stores/graph.ts

```diff
@@ -5,46 +5,89 @@ import { Vec2 } from '@/util/vec2'
 import { assertNever, assert } from '@/util/assert'
 import { useProjectStore } from './project'
 import * as Y from 'yjs'
-import { useObserveYjs, useObserveYjsDeep } from '@/util/crdt'
+import { useObserveYjs } from '@/util/crdt'
 import {
-  rangeEncloses,
   type ContentRange,
   type ExprId,
   type IdMap,
   type NodeMetadata,
-} from '../../shared/yjs-model'
+  rangeIntersects,
+} from 'shared/yjs-model'
 import type { Opt } from '@/util/opt'
+import { parseEnso } from '@/util/ffi'
 
 export const useGraphStore = defineStore('graph', () => {
   const proj = useProjectStore()
 
   proj.setProjectName('test')
   proj.setObservedFileName('Main.enso')
 
-  let text = computed(() => proj.module?.contents)
-  let metadata = computed(() => proj.module?.metadata)
+  const text = computed(() => proj.module?.contents)
```

Comment:
not familiar enough with vue to know whether this is necessary - does reactivity not work properly if you just do `const text = proj.module?.contents`?

Reply:
No. If the module changes, your local `text` variable would not be updated accordingly.

Reply:
oh right, `text` is supposed to be a `Ref`